spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -239,3 +239,40 @@ The default Task/Batch event and destination names on the broker are enumerated
 |Item Write events|`item-write-events`
 |Skip events|`skip-events`
 |===
+
+[[spring-cloud-dataflow-launch-tasks-from-stream]]
+== Launching Tasks from a Stream
+
+You can launch a task from a stream by using one of the available `task-launcher` sinks. Currently the only available
+`task-launcher` sink is the `task-launcher-local` which will launch a task on your local machine.
+
+NOTE: `task-launcher-local` is meant for development purposes only.
+
+A `task-launcher` sink expects a message containing a TaskLaunchRequest object in its payload. From the
+TaskLaunchRequest object the task-launcher will obtain the URI of the artifact to be launched as well as the
+properties and command line arguments to be used by the task.
+
+The `task-launcher-local` can be added to the available sinks by executing the app register command as follows:
+
+```
+app register --name task-launcher-local --type sink --uri maven://org.springframework.cloud.stream.app:task-launcher-local-sink-kafka:jar:1.0.0.BUILD-SNAPSHOT
+```
+
+=== TriggerTask
+
+The easiest way to launch a task using the `task-launcher` is to use the `triggertask` source. The `triggertask` source
```

Comment:
I'm not sure this is actually "easier" than just doing `task launch foo` from a shell.